src/services/poolTogether.js

```diff
@@ -365,7 +365,8 @@ export async function getPoolTogetherTransactions(symbol: string, address: strin
   const { unitType, poolContractAddress: contractAddress } = getPoolTogetherTokenContract(symbol);
   let deposits = [];
   let withdrawals = [];
-  const rawHistory = await fetchPoolTogetherHistory(contractAddress, address);
+  const rawHistory = await fetchPoolTogetherHistory(contractAddress, address)
+    .catch(() => null);
```

Comment:
Could we report this at least and then return `null`?

Reply:
It's already reported in `callSubgraph`. This is one of the places where we don't want any additional handling -- that is, no changes to the UI, so it follows the previous behaviour of `callSubgraph` (returning `null` on error), only a bit higher in the call stack.